Yoast/javascript
# Add sentence beginnings assessment for Hebrew

## Summary

This PR can be summarized in the following changelog entry: 

* [yoastseo] Adds the consecutive sentences assessment for Hebrew.
* [Yoast SEO] Adds the consecutive sentences assessment for Hebrew.

## Test instructions

This PR can be tested by following these steps:
1. Run yarn test and make sure all current specs pass.
2. In the content analysis app set the language to Hebrew (he_IL)
* Write three or more Hebrew sentences...
    * that all begin with the same written-out number, like `אחד` ("one" in masc. form)
       * E.g. `.אחד פרי מונח על השולחן. אחד פירות על השולחן. אחד פירות על השולחן` 
    * that all begin with the same demonstrative pronoun, like `זה` ('this').
       * E.g. `.זה המסמך מוכן. זה המשחק נגמר. זה הסרט התחיל`
* In each of the above cases, the sentence beginning readability assessment should read: `Consecutive sentences: There is enough variety in your sentences. That's great!`
* Write three or more Hebrew sentences that begin with the same word.
  * E.g. `.שלום מה שלומך? שלום אני בסדר. שלום עולם! שלום לכל אחד מכם`
* The sentence beginning readability assessment should read: `Consecutive sentences: The text contains 4 consecutive sentences starting with the same word. Try to mix things up!`

## Quality assurance


Fixes #https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-514
